### ISSUE TITLE
6.0 client XPack plugins don't work?

### ISSUE THREAD

**Issue by warewolf** [elastic/elasticsearch-perl#179]
6.0 version of both Search::Elasticsearch and Search::Elasticsearch::Plugin::XPack are installed

```
$ perl -MSearch::Elasticsearch\ 99999 -e 1
Search::Elasticsearch version 99999 required--this is only version 6.00.
BEGIN failed--compilation aborted.

$ perl -MSearch::Elasticsearch::Plugin::XPack\ 99999 -e 1
Search::Elasticsearch::Plugin::XPack version 99999 required--this is only version 6.00.
BEGIN failed--compilation aborted.
$ 
```

Test script:
```
#!/usr/bin/env perl

use strict;
use warnings;
use Search::Elasticsearch;

my $ES = Search::Elasticsearch->new( # {{{
  nodes => [ 'localhost' ],
  plugins => ['XPack'],
); # }}}

my $response = $ES->xpack->security->put_role_mapping(
  name => 'test',
  body => { some => 'thing' },
);
```

Result:
```
Can't locate object method "xpack" via package "Search::Elasticsearch::Client::6_0::Direct__WITH__Search::Elasticsearch::Plugin::XPack::6_0" at test.pl line 12.
```

I don't grok Moo/Moose and roles, but it appears that the _builder for the security stuff somehow isn't getting called, and the accessor method `->xpack` isn't getting added to `Search::Elasticsearch`?

**Comment by clintongormley** [comment]
Oops - I messed this up (a long time ago).

Just drop the `->xpack` part, so you can do

```
$ES->security->put_role_mapping...
```

The plan was to drop the x-pack bit in the next version anyway. I seem to have done it inadvertently. Apologies for that.

**Comment by warewolf** [comment]
That fixes it, thank you!

**Comment by ezimuel** [comment]
@warewolf I just released a **dev version** [6.80_1](https://github.com/elastic/elasticsearch-perl/releases/tag/version_6.80_1) that updated all the endpoints with the latest Elasticsearch [6.8.7](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/release-notes-6.8.7.html). I fixed the issue about XPack, now there's an `xpack()` object. Moreover, I merged the XPack endpoints into the `Direct` client so you don't need anymore to use a plugins for that. I leaved the XPack.pm as NOOP for backward compatibility.
Can you try this **dev release** and let me know? Thanks!

**Comment by ezimuel** [comment]
I just released [version 6.80](https://github.com/elastic/elasticsearch-perl/releases/tag/version_6.80)
